pages/common/git-rm.md

```diff
@@ -0,0 +1,15 @@
+# git rm
+
+> Remove files from the working tree and from the index
+
+- Remove file and add its deletion to next commit
+
+`git rm {{file}}`
+
+- Remove file from all previous commits but keep it locally
```

Comment:
Is this accurate? Does this actually rewrite history by removing the file from previous commits?? I'm pretty sure it doesn't.

My proposal:
- git rm: "Remove file from repository (stop tracking it, will appear as a deletion in the next commit) and actually delete it in the local filesystem"
- git rm --cached: "Remove file from repository (stop tracking it, will appear as a deletion in the next commit) but keep it untouched it in the local filesystem"

Git experts: is this accurate?